2020/08/11 15:51:28 [emerg] 1#1: upstream "afisha_nuxt" may not have port 3000 in /etc/nginx/conf.d/default.conf:49
2020/08/11 15:53:43 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.27.0.3:3000/", host: "localhost"
2020/08/11 15:53:43 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.27.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/"
2020/08/11 15:53:50 [error] 28#28: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.27.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 16:06:56 [error] 28#28: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.27.0.3:3000/", host: "0.0.0.0"
2020/08/11 16:06:56 [error] 28#28: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.27.0.3:3000/favicon.ico", host: "0.0.0.0", referrer: "http://0.0.0.0/"
2020/08/11 16:07:00 [error] 28#28: *19 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.27.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 16:12:28 [error] 28#28: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.27.0.3:3000/", host: "0.0.0.0"
2020/08/11 16:12:28 [error] 28#28: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.27.0.3:3000/favicon.ico", host: "0.0.0.0", referrer: "http://0.0.0.0/"
2020/08/11 16:24:42 [error] 28#28: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 16:39:58 [error] 28#28: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 16:49:39 [error] 27#27: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 16:49:46 [error] 27#27: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 16:50:26 [error] 27#27: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /static/admin HTTP/1.1", upstream: "http://172.28.0.3:3000/var/www/static", host: "localhost"
2020/08/11 16:50:26 [error] 27#27: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/admin"
2020/08/11 16:51:24 [error] 27#27: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/admin/img/icon-alert.svg"
2020/08/11 16:53:10 [error] 27#27: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /static/ HTTP/1.1", upstream: "http://172.28.0.3:3000/var/www/static", host: "localhost"
2020/08/11 16:53:10 [error] 27#27: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/"
2020/08/11 16:53:30 [error] 27#27: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/favicon.ico"
2020/08/11 16:55:19 [error] 27#27: *25 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/"
2020/08/11 16:55:20 [error] 27#27: *25 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/admin/img/icon-alert.svg"
2020/08/11 16:55:21 [error] 27#27: *25 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/admin"
2020/08/11 16:55:23 [error] 27#27: *25 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 16:55:24 [error] 27#27: *35 connect() failed (111: Connection refused) while connecting to upstream, client: 172.28.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.28.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:24:06 [error] 27#27: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.32.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.32.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:25:53 [error] 27#27: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.32.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.32.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:27:14 [error] 27#27: *22 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.32.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.32.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:28:35 [error] 28#28: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.48.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:30:17 [error] 28#28: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.48.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:30:48 [error] 28#28: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.48.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:34:52 [error] 28#28: *22 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.48.3:3000/favicon.ico", host: "0.0.0.0", referrer: "http://0.0.0.0/static/favicon.ico"
2020/08/11 18:35:05 [error] 28#28: *22 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.48.3:3000/favicon.ico", host: "0.0.0.0", referrer: "http://0.0.0.0/"
2020/08/11 18:57:46 [error] 30#30: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.64.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.64.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:57:48 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.64.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.64.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:58:58 [error] 27#27: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.80.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.80.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:59:38 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.80.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.80.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 18:59:49 [error] 27#27: *19 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.80.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.80.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 19:00:45 [error] 27#27: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.80.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.80.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 19:13:08 [emerg] 1#1: host not found in upstream "project:8000" in /etc/nginx/conf.d/default.conf:3
2020/08/11 19:13:36 [error] 27#27: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.96.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 19:14:34 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.96.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 19:14:53 [error] 27#27: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.96.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 19:15:46 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.96.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 19:19:58 [error] 27#27: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 19:22:13 [error] 27#27: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 19:22:46 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/login/?next=/admin/"
2020/08/11 19:23:07 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/var/www/static", host: "localhost"
2020/08/11 19:23:07 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/favicon.ico"
2020/08/11 19:23:36 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/var/www/static", host: "localhost"
2020/08/11 19:23:36 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/favicon.ico"
2020/08/11 19:23:49 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/var/www/static", host: "localhost"
2020/08/11 19:23:49 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/favicon.ico"
2020/08/11 19:23:50 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/var/www/static", host: "localhost"
2020/08/11 19:23:50 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/favicon.ico"
2020/08/11 19:24:12 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/var/www/static", host: "localhost"
2020/08/11 19:24:13 [error] 27#27: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.112.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/static/favicon.ico"
2020/08/12 11:22:27 [emerg] 1#1: "alias" directive is duplicate in /etc/nginx/conf.d/default.conf:54
2020/08/12 12:05:36 [emerg] 1#1: "alias" directive is duplicate in /etc/nginx/conf.d/default.conf:54
2020/08/12 12:19:33 [emerg] 1#1: "alias" directive is duplicate in /etc/nginx/conf.d/default.conf:54
2020/08/12 12:20:29 [emerg] 1#1: "alias" directive is duplicate in /etc/nginx/conf.d/default.conf:54
2020/08/12 12:21:22 [emerg] 1#1: invalid number of arguments in "root" directive in /etc/nginx/conf.d/default.conf:54
2020/08/12 12:24:33 [emerg] 1#1: invalid number of arguments in "root" directive in /etc/nginx/conf.d/default.conf:54
2020/08/12 12:25:13 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /robots.txt HTTP/1.1", upstream: "http://172.21.0.3:3000/robots.txt", host: "0.0.0.0"
2020/08/12 12:25:13 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.21.0.3:3000/favicon.ico", host: "0.0.0.0", referrer: "http://0.0.0.0/static/favicon.ico"
2020/08/12 12:25:23 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /admin HTTP/1.1", upstream: "http://172.21.0.3:3000/admin", host: "0.0.0.0"
2020/08/12 12:25:23 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.21.0.3:3000/favicon.ico", host: "0.0.0.0", referrer: "http://0.0.0.0/admin"
2020/08/12 12:25:56 [error] 29#29: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /admin HTTP/1.1", upstream: "http://172.21.0.3:3000/admin", host: "localhost"
2020/08/12 12:25:56 [error] 29#29: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.21.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin"
2020/08/12 12:57:45 [emerg] 1#1: host not found in upstream "backend:8000" in /etc/nginx/conf.d/default.conf:63
2020/08/12 15:06:41 [emerg] 1#1: upstream "afisha_nuxt" may not have port 3000 in /etc/nginx/conf.d/default.conf:40
2020/08/12 15:08:08 [emerg] 1#1: upstream "afisha_nuxt" may not have port 3000 in /etc/nginx/conf.d/default.conf:40
2020/08/12 15:09:42 [emerg] 1#1: upstream "afisha_nuxt" may not have port 3000 in /etc/nginx/conf.d/default.conf:40
2020/08/12 15:10:26 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:49
2020/08/12 15:13:36 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.31.0.3:3000/", host: "localhost"
2020/08/12 15:13:36 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.31.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/"
2020/08/12 15:16:47 [error] 27#27: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.31.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/"
2020/08/12 15:18:20 [error] 27#27: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.31.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/admin/"
2020/08/12 15:18:50 [error] 27#27: *16 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.31.0.3:3000/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/admin/login/?next=/admin/"
2020/08/12 15:19:20 [error] 27#27: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.31.0.3:3000/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/admin/login/?next=/admin/"
2020/08/12 15:19:23 [error] 27#27: *16 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.31.0.3:3000/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/admin/login/?next=/admin/"
2020/08/12 15:19:35 [error] 27#27: *16 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.31.0.3:3000/", host: "127.0.0.1"
2020/08/12 15:19:35 [error] 27#27: *16 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.31.0.3:3000/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2020/08/12 15:23:42 [emerg] 1#1: upstream "afisha_nuxt" may not have port 3000 in /etc/nginx/conf.d/default.conf:40
2020/08/12 15:27:55 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://192.168.48.3:3000/", host: "127.0.0.1"
2020/08/12 15:27:55 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.48.3:3000/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2020/08/12 15:32:49 [error] 29#29: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://192.168.48.3:3000/", host: "127.0.0.1"
2020/08/12 15:32:50 [error] 29#29: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.48.3:3000/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2020/08/12 16:11:24 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.64.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://192.168.64.3:3000/", host: "127.0.0.1"
2020/08/12 16:11:24 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.64.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.64.3:3000/favicon.ico", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2020/08/12 16:11:31 [error] 27#27: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.64.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://192.168.64.3:3000/", host: "0.0.0.0"
2020/08/12 16:11:31 [error] 27#27: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.64.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.64.3:3000/favicon.ico", host: "0.0.0.0", referrer: "http://0.0.0.0/"
2020/08/12 16:11:39 [error] 27#27: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.64.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://192.168.64.3:3000/", host: "localhost"
2020/08/12 16:11:39 [error] 27#27: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.64.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://192.168.64.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/"
2020/08/13 10:25:29 [emerg] 1#1: host not found in upstream "afisha_nuxt:3000" in /etc/nginx/conf.d/default.conf:7
2020/08/13 11:56:27 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:47
2020/08/13 12:17:38 [emerg] 1#1: the "alias" directive cannot be used inside the named location in /etc/nginx/conf.d/default.conf:48
2020/08/13 12:18:09 [emerg] 1#1: invalid number of arguments in "try_files" directive in /etc/nginx/conf.d/default.conf:48
2020/08/13 12:22:56 [error] 27#27: *1 open() "/project/static/static/favicon.ico" failed (2: No such file or directory), client: 192.168.80.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", host: "localhost"
2020/08/13 12:23:19 [error] 27#27: *1 open() "/project/static/static/favicon.ico" failed (2: No such file or directory), client: 192.168.80.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", host: "localhost"
2020/08/13 12:23:30 [error] 27#27: *1 open() "/project/static/static/favicon.ico" failed (2: No such file or directory), client: 192.168.80.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", host: "localhost"
2020/08/13 12:24:16 [error] 27#27: *1 open() "/project/static/static/favicon.ico" failed (2: No such file or directory), client: 192.168.80.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", host: "localhost"
2020/08/13 12:24:20 [error] 27#27: *1 "/project/static/static/index.html" is not found (2: No such file or directory), client: 192.168.80.1, server: localhost, request: "GET /static/ HTTP/1.1", host: "localhost"
2020/08/13 12:25:50 [error] 27#27: *1 "/project/static/static/index.html" is not found (2: No such file or directory), client: 192.168.80.1, server: localhost, request: "GET /static/ HTTP/1.1", host: "localhost"
2020/08/13 12:25:53 [error] 27#27: *1 open() "/project/static/static/favicon.ico" failed (2: No such file or directory), client: 192.168.80.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", host: "localhost"
2020/08/13 12:26:34 [emerg] 1#1: "try_files" directive is duplicate in /etc/nginx/conf.d/default.conf:45
2020/08/13 12:28:57 [error] 27#27: *5 open() "/project/static/static/favicon.ico" failed (2: No such file or directory), client: 192.168.80.1, server: localhost, request: "GET /static/favicon.ico HTTP/1.1", host: "localhost"
2020/08/31 10:22:45 [warn] 27#27: *1 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.21.0.1, server: localhost, request: "GET /api/v1/sessions/1/ HTTP/1.1", upstream: "http://172.21.0.4:8000/api/v1/sessions/1/", host: "127.0.0.1"
2020/08/31 11:47:13 [warn] 30#30: *14 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.22.0.1, server: localhost, request: "GET /api/v1/sessions/1/ HTTP/1.1", upstream: "http://172.22.0.4:8000/api/v1/sessions/1/", host: "127.0.0.1"
2020/09/07 10:27:19 [emerg] 1#1: host not found in upstream "project:8000" in /etc/nginx/conf.d/default.conf:3
2020/09/07 10:27:25 [emerg] 1#1: host not found in upstream "project:8000" in /etc/nginx/conf.d/default.conf:3
2020/09/07 10:27:29 [emerg] 1#1: host not found in upstream "project:8000" in /etc/nginx/conf.d/default.conf:3
2020/09/07 13:32:12 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:32:14 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:32:16 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:32:17 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:32:24 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:32:28 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:32:33 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:32:40 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:32:54 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:38:26 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:38:28 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:38:30 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:38:31 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:38:33 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:38:36 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:38:43 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:38:56 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:52
2020/09/07 13:39:10 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:39:23 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:39:27 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:39:29 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:39:31 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:39:33 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:39:37 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:39:41 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:39:49 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:40:03 [emerg] 1#1: unexpected "}" in /etc/nginx/conf.d/default.conf:54
2020/09/07 13:42:56 [warn] 27#27: *1 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.20.0.1, server: localhost, request: "GET /adm12345622UUjjsanjaooJjf5235/ HTTP/1.1", upstream: "http://172.20.0.4:8000/adm12345622UUjjsanjaooJjf5235/", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/"
2020/09/07 14:11:51 [error] 29#29: *54 open() "/project/media/media/images/image_41" failed (2: No such file or directory), client: 172.20.0.1, server: localhost, request: "GET /media/images/image_41 HTTP/1.1", host: "127.0.0.1"
2020/09/07 14:12:09 [error] 29#29: *54 open() "/project/media/media/images/image_41.jpg" failed (2: No such file or directory), client: 172.20.0.1, server: localhost, request: "GET /media/images/image_41.jpg HTTP/1.1", host: "127.0.0.1"
2020/09/07 15:35:13 [error] 28#28: *10 open() "/var/www/media/media/images/image_7.jpg" failed (2: No such file or directory), client: 172.20.0.1, server: localhost, request: "GET /media/images/image_7.jpg HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/films/film/"
2020/09/10 09:45:04 [error] 28#28: *1 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/10 09:45:04 [error] 28#28: *5 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/10 09:45:05 [error] 28#28: *6 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/10 09:45:05 [error] 28#28: *10 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/10 09:45:05 [error] 28#28: *9 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/static/admin/css/fonts.css"
2020/09/10 09:45:05 [error] 28#28: *8 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/static/admin/css/fonts.css"
2020/09/10 09:45:05 [error] 28#28: *11 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/static/admin/css/fonts.css"
2020/09/11 09:25:51 [error] 27#27: *10 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/css/base.css HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:51 [error] 27#27: *7 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:51 [error] 27#27: *6 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:51 [error] 27#27: *8 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:51 [error] 27#27: *3 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/css/responsive.css HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:51 [error] 27#27: *12 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:51 [error] 27#27: *17 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:51 [error] 27#27: *18 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:51 [error] 27#27: *19 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/css/responsive.css HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:53 [error] 27#27: *20 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/css/base.css HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:53 [error] 27#27: *22 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/css/responsive.css HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:53 [error] 27#27: *24 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:53 [error] 27#27: *25 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:53 [error] 27#27: *26 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:53 [error] 27#27: *23 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:53 [error] 27#27: *28 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:53 [error] 27#27: *29 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:25:53 [error] 27#27: *30 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /static/admin/css/responsive.css HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/django_celery_beat/periodictask/1/change/"
2020/09/11 09:28:16 [error] 27#27: *33 rewrite or internal redirection cycle while redirect to named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /robots.txt HTTP/1.1", host: "127.0.0.1"
2020/09/11 09:29:56 [emerg] 1#1: "try_files" directive is duplicate in /etc/nginx/conf.d/default.conf:45
2020/09/11 09:30:00 [emerg] 1#1: "try_files" directive is duplicate in /etc/nginx/conf.d/default.conf:45
2020/09/11 09:30:02 [emerg] 1#1: "try_files" directive is duplicate in /etc/nginx/conf.d/default.conf:45
2020/09/11 09:30:05 [emerg] 1#1: "try_files" directive is duplicate in /etc/nginx/conf.d/default.conf:45
2020/09/11 09:30:08 [emerg] 1#1: "try_files" directive is duplicate in /etc/nginx/conf.d/default.conf:45
2020/09/11 09:30:14 [emerg] 1#1: host not found in upstream "project:8000" in /etc/nginx/conf.d/default.conf:3
2020/09/11 09:31:16 [error] 27#27: *1 could not find named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /robots.txt HTTP/1.1", host: "127.0.0.1"
2020/09/11 09:31:17 [error] 27#27: *2 could not find named location "@second_static", client: 172.18.0.1, server: localhost, request: "GET /robots.txt HTTP/1.1", host: "127.0.0.1"
2020/09/11 11:06:10 [warn] 27#27: *15 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000001, client: 172.18.0.1, server: localhost, request: "POST /adm12345622UUjjsanjaooJjf5235/films/cinema/3/change/ HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/films/cinema/3/change/"
2020/09/11 11:07:13 [warn] 27#27: *14 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000002, client: 172.18.0.1, server: localhost, request: "POST /adm12345622UUjjsanjaooJjf5235/films/cinema/4/change/ HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/films/cinema/4/change/"
2020/09/11 11:11:11 [warn] 27#27: *71 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000003, client: 172.18.0.1, server: localhost, request: "POST /adm12345622UUjjsanjaooJjf5235/films/cinema/1/change/ HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/films/cinema/1/change/"
2020/09/13 20:27:04 [warn] 28#28: *6 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000001, client: 172.18.0.1, server: localhost, request: "POST /adm12345622UUjjsanjaooJjf5235/films/cinema/4/change/ HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/films/cinema/4/change/"
2020/09/13 20:27:14 [warn] 28#28: *6 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000002, client: 172.18.0.1, server: localhost, request: "POST /adm12345622UUjjsanjaooJjf5235/films/cinema/3/change/ HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/films/cinema/3/change/"
2020/09/13 20:27:38 [warn] 28#28: *6 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000003, client: 172.18.0.1, server: localhost, request: "POST /adm12345622UUjjsanjaooJjf5235/films/cinema/2/change/ HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/films/cinema/2/change/"
2020/09/13 20:27:47 [warn] 28#28: *6 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000004, client: 172.18.0.1, server: localhost, request: "POST /adm12345622UUjjsanjaooJjf5235/films/cinema/1/change/ HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/adm12345622UUjjsanjaooJjf5235/films/cinema/1/change/"
2020/09/15 08:54:06 [warn] 1#1: server name "http://www.kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:31
2020/09/15 08:56:13 [warn] 1#1: server name "http://www.kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:31
2020/09/15 08:58:37 [warn] 1#1: server name "http://www.kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:31
2020/09/15 09:13:05 [warn] 1#1: server name "http://www.kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:31
2020/09/15 09:19:58 [warn] 1#1: server name "http://www.kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:31
2020/09/15 09:23:35 [warn] 1#1: server name "http://www.kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:31
2020/09/15 09:33:08 [warn] 1#1: server name "http://www.kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:35
2020/09/15 09:34:44 [warn] 1#1: server name "http://kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:35
2020/09/15 09:40:21 [warn] 42#42: server name "http://kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:35
2020/09/15 09:40:22 [warn] 1#1: server name "http://kino-39.ru/" has suspicious symbols in /etc/nginx/conf.d/default.conf:35
2020/09/15 09:54:13 [emerg] 1#1: unexpected ";" in /etc/nginx/conf.d/default.conf:33
2020/09/15 09:54:15 [emerg] 1#1: unexpected ";" in /etc/nginx/conf.d/default.conf:33
2020/09/15 09:54:18 [emerg] 1#1: unexpected ";" in /etc/nginx/conf.d/default.conf:33
2020/09/15 09:54:20 [emerg] 1#1: unexpected ";" in /etc/nginx/conf.d/default.conf:33
2020/09/15 09:54:22 [emerg] 1#1: unexpected ";" in /etc/nginx/conf.d/default.conf:33
2020/09/15 09:54:25 [emerg] 1#1: unexpected ";" in /etc/nginx/conf.d/default.conf:33
2020/09/15 09:54:31 [emerg] 1#1: unexpected ";" in /etc/nginx/conf.d/default.conf:33
2020/09/15 09:54:38 [emerg] 1#1: unexpected ";" in /etc/nginx/conf.d/default.conf:33
2020/09/15 10:04:11 [error] 27#27: *1 could not find named location "@second_static", client: 5.43.243.19, server: www.kino-39.ru, request: "GET /static/rest_framework/css/bootstrap.min.css.map HTTP/1.1", host: "www.kino-39.ru"
2020/09/15 11:42:06 [notice] 28#28: signal process started
2020/09/15 11:51:01 [notice] 10#10: signal process started
2020/09/15 11:58:08 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:58:10 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:58:11 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:58:13 [emerg] 8#8: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:58:15 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:58:18 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:58:22 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:58:29 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:58:43 [emerg] 8#8: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:59:31 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:59:33 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:59:35 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 11:59:38 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:00:57 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:01:00 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:01:03 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:01:06 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:01:08 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:01:11 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:01:16 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:01:24 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:01:37 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:02:04 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:02:57 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:03:58 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:04:59 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:00 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:17 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:20 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:24 [emerg] 8#8: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:26 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:29 [emerg] 8#8: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:31 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:35 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:43 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:06:56 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:07:23 [emerg] 7#7: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:07:31 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:07:34 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:07:35 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:07:37 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:07:40 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:07:43 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:07:48 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:07:56 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:08:10 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:08:36 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:09:28 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:10:29 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:11:31 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:12:32 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:13:33 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:14:34 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:15:35 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:16:36 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:17:37 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:18:38 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:19:40 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:20:41 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:21:42 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:22:43 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:23:44 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:24:45 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:25:46 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:26:47 [emerg] 1#1: open() "/etc/letsencrypt/options-ssl-nginx.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:41
2020/09/15 12:27:14 [emerg] 1#1: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 12:27:18 [emerg] 1#1: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 12:27:21 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:27:24 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:27:26 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:27:29 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:27:33 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:27:41 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:27:55 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:28:21 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:29:13 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:30:15 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:31:16 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:32:17 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:33:18 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:34:19 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:35:20 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:42:10 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 12:42:14 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 12:42:19 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:42:21 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:42:23 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:42:26 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:42:31 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:42:38 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:42:52 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:43:18 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:44:11 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 12:45:12 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:07:48 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 13:07:51 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 13:07:55 [emerg] 8#8: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 13:07:58 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:08:01 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:08:04 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:08:08 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:08:15 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:08:29 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:08:55 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:09:48 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:10:49 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:11:50 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:12:51 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:13:52 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:14:53 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:15:55 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:16:56 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:17:57 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:18:58 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:19:59 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:21:00 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:22:01 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:40:37 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:40:39 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:40:41 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:40:43 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:40:46 [emerg] 8#8: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:40:50 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:40:54 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:43:53 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 13:43:56 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 13:44:00 [emerg] 8#8: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:44:03 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:44:06 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:44:09 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:44:13 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:44:21 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:44:34 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:45:01 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:45:53 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:46:54 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:47:56 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:48:57 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:49:58 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:50:59 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:52:00 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:53:01 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:54:02 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:55:04 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:56:05 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:57:06 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:58:07 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 13:59:08 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:00:10 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:01:11 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:02:12 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:03:13 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:03:51 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:03:53 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:03:55 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:03:57 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:04:00 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:04:04 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:04:08 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:04:16 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:04:30 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:04:56 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:05:49 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:10:06 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:10:08 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:10:10 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:10:12 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:10:14 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:10:16 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:10:20 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:10:28 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:10:43 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:11:09 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:12:02 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:13:03 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:14:04 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:15:05 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:16:06 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:17:07 [emerg] 7#7: SSL_CTX_use_certificate("/etc/letsencrypt/live/kino-39.ru/fullchain.pem") failed (SSL: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small)
2020/09/15 14:27:39 [emerg] 10#10: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:34:59 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:35:01 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:35:03 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:35:05 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:35:08 [emerg] 8#8: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:35:11 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:35:15 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:35:23 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:35:36 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:36:03 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:36:55 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:37:56 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:38:57 [emerg] 7#7: cannot load certificate "/etc/letsencrypt/live/kino-39.ru/fullchain.pem": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/letsencrypt/live/kino-39.ru/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2020/09/15 14:40:22 [notice] 10#10: signal process started
2020/09/15 14:47:58 [notice] 10#10: signal process started
2020/09/15 15:45:30 [error] 9#9: *40 could not find named location "@second_static", client: 5.43.243.19, server: kino-39.ru, request: "GET /robots.txt HTTP/1.1", host: "kino-39.ru"
2020/09/15 15:47:06 [error] 9#9: *47 could not find named location "@second_static", client: 5.43.243.19, server: kino-39.ru, request: "GET /robots.txt HTTP/1.1", host: "kino-39.ru"
2020/09/15 15:48:05 [error] 9#9: *1 rewrite or internal redirection cycle while internally redirecting to "/var/www/static", client: 5.43.243.19, server: kino-39.ru, request: "GET /robots.txt HTTP/1.1", host: "kino-39.ru"
2020/09/15 15:48:40 [error] 9#9: *2 rewrite or internal redirection cycle while internally redirecting to "/var/www/static", client: 5.43.243.19, server: kino-39.ru, request: "GET /robots.txt HTTP/1.1", host: "kino-39.ru"
2020/09/15 15:48:41 [error] 9#9: *4 rewrite or internal redirection cycle while internally redirecting to "/var/www/static", client: 5.43.243.19, server: kino-39.ru, request: "GET /robots.txt HTTP/1.1", host: "kino-39.ru"
2020/09/15 15:48:42 [error] 9#9: *6 rewrite or internal redirection cycle while internally redirecting to "/var/www/static", client: 5.43.243.19, server: kino-39.ru, request: "GET /robots.txt HTTP/1.1", host: "kino-39.ru"
2020/09/15 15:53:01 [emerg] 7#7: unknown directive "root/var/www/static" in /etc/nginx/conf.d/default.conf:69
2020/09/15 15:53:03 [emerg] 7#7: unknown directive "root/var/www/static" in /etc/nginx/conf.d/default.conf:69
2020/09/15 15:53:06 [emerg] 7#7: unknown directive "root/var/www/static" in /etc/nginx/conf.d/default.conf:69
2020/09/15 15:53:08 [emerg] 8#8: unknown directive "root/var/www/static" in /etc/nginx/conf.d/default.conf:69
2020/09/15 15:53:11 [emerg] 8#8: unknown directive "root/var/www/static" in /etc/nginx/conf.d/default.conf:69
2020/09/15 15:53:15 [emerg] 7#7: unknown directive "root/var/www/static" in /etc/nginx/conf.d/default.conf:69
2020/09/15 15:53:21 [emerg] 7#7: unknown directive "root/var/www/static" in /etc/nginx/conf.d/default.conf:69
2020/09/15 15:53:28 [emerg] 7#7: unknown directive "root/var/www/static" in /etc/nginx/conf.d/default.conf:69
2020/09/15 16:02:46 [error] 9#9: *48 rewrite or internal redirection cycle while internally redirecting to "/var/www/static", client: 5.43.243.19, server: kino-39.ru, request: "GET /static/rest_framework/css/bootstrap.min.css.map HTTP/1.1", host: "kino-39.ru"
